### DISCUSSION TITLE
Designing Mechanics: Plants

### DISCUSSION THREAD

**Discussion by wallamide** [Leafwing-Studios/Emergence#144] [Design notes]
# Making an "Organic" Factory Sim

As we lay the framework for basic game functionality, we need to consider ways to use biological systems to inspire our game mechanics. Here is a discussion to start planning some mechanics for plants and species we may wish to mimic/use. 

## Current plans

Currently we plan for plants to produce energy in the form of biomass that ants collect and give to fungi to maintain the fungi and receive useable food. Our demo steam page also calls for metal extracting plants. 

## Potential Starting Points

Plants fill many roles in their many ecosystems. Most have a role in planetary cycles through things like: 

- nitrogen fixation, 
- carbon sequestration, 
- water filtration,  
- metal fixation, and
- air filtration. 

On top of this, many produce pheromones that allow for the attraction of animals of various kinds.

**Comment by alice-i-cecile** [comment]
# Fiber Production

Fiber is an interesting material and resource, and there are a large number of potential plants (and animals even) that could produce this.

**Comment by alice-i-cecile** [comment]
# Defenses

Carnivorous plants, thorns and poisonous plants are great potential defense mechanisms.

**Comment by bzm3r** [comment]
Closely related to: #111

**Comment by wallamide** [comment]
# Calling for help

Cultivate plants that have mutualistic relationships with animals to attract them to where the colony is, then work with the new animals to accomplish tasks.

## Repulsion

Some plants also disgust some animals. Cultivating them helps prevent attacks or reduces the chances of encountering some animals.

These plant mechanics will also give us ways to use the pheromone system with plants.

**Comment by wallamide** [comment]
# Plant-Animal-Hive Mind Mutualism

Cultivating some plants could require the player to form a mutual relationship with some animal that can pollinate certain types of plants that then need these animals living and working with the hive mind to maintain these plants.
